### PR TITLE
Fix for https://github.com/red-hat-storage/ocs-ci/issues/386

### PR DESCRIPTION
**Pull request by RazTamir**

### CODE REVIEW

File: tests/manage/test_pvc_deletion_during_io.py [hunk 1]
```diff
@@ -52,6 +52,13 @@ def test_run_io_and_delete_pvc(self):
             if "NotFound" in str(ex):
                 pass
 
+
+@pytest.mark.usefixtures(delete_pod.__name__)
+class TestRunningIO(TestDeletePVCWhileRunningIO):
```
Comment (ebenahar) [inline tests/manage/test_pvc_deletion_during_io.py]: I don't think we should inherit from the test class.
Let's create a base class which the basic fixtures and inherit from it for both TCs

Comment (RazTamir) [reply tests/manage/test_pvc_deletion_during_io.py]: I agree.
Will fix it

File: tests/fixtures.py [hunk 2]
```diff
@@ -144,3 +144,16 @@ def create_rbd_pod(request):
         interface_type=constants.CEPHBLOCKPOOL,
         pvc_name=class_instance.pvc_obj.name
     )
+
+
+@pytest.fixture()
```
Comment (petr-balogh) [inline tests/fixtures.py]: I think this is duplicate work of those two PRs:
https://github.com/red-hat-storage/ocs-ci/pull/396/files#diff-8a66b2566ee3ea0285647f603db9174bR159
https://github.com/red-hat-storage/ocs-ci/pull/339/files#diff-10993bfc055d543860f899ca9733c55aR20

**Comment by ebenahar** [comment]
Not needed anymore, the test will be removed with https://github.com/red-hat-storage/ocs-ci/pull/435

**Pull request closed without merging**